Rename PKI to Identity Issuance (#6972)
istio/istio.io: content/en/docs/concepts/security/index.md

   service account refers to the existing service account just like the
   identities that the customer's Identity Directory manages.

> ## Identity Issuance {#pki}

Reviewer: ```suggestion
## Identity and Certificate Management {#pki}
```

Reviewer 2: Perhaps we should also change this "pki" reference?  Maybe "identity"?  If so, we could also remove the lower-case "pki" definition from the ".spelling" file that was recently added.

Reviewer 3: Make "certificate management" lowercase. (Unless it's a product name or a trademarked term or something like that.) (See https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings_1)

Reviewer 4: ```suggestion
## Identity and certificate management {#pki}
```

> The Istio securely provisions strong identities

Reviewer: ```suggestion
Istio securely provisions strong identities
```

> to every workload with X.509 certificates. The Istio agent running alongside 

Reviewer: ```suggestion
to every workload with X.509 certificates. The istiod and istio-agents running alongside 
```

Reviewer 2: It looks like elsewhere, we use backticks around "istiod".  Do you think it should be done here, too?

Reviewer 4: ```suggestion
to every workload with X.509 certificates. Istio agents, running alongside each Envoy proxy, 
```

> to every workload with X.509 certificates. Istio agents, running alongside each Envoy proxy, 

Reviewer 4: ```suggestion
to every workload with X.509 certificates. Istio agents, running alongside each Envoy proxy,
```

> each Envoy proxy automates key and certificate

Reviewer 4: ```suggestion
work together with `istiod` to automate key and certificate
```

Istio provisions identities through the secret discovery service (SDS) using the
following flow:

1. `istiod` offers a gRPC service to take [certificate signing requests](https://en.wikipedia.org/wiki/Certificate_signing_request) (CSRs).
1. Envoy sends a certificate and key request via the Envoy secret discovery
   service (SDS) API.
1. Upon receiving the SDS request, the Istio agent creates the private key

>    and CSR before sending the CSR with its credentials to the `istiod` for signing.

Reviewer 2: ```suggestion
   and CSR before sending the CSR with its credentials to `istiod` for signing.
```

mechanisms. The control plane may fetch the public key and attach it to the
configuration for JWT validation. Alternatively, Istiod provides the path to the
keys and certificates the Istio system manages and installs them to the

> application pod for mutual TLS. You can find more info in the [Identity Issuance section](/docs/concepts/security/#pki).

Reviewer: ```suggestion
application pod for mutual TLS. You can find more info in the [Identity and Certificate Management section](/docs/concepts/security/#pki).
```

Reviewer 4: Also no need for path.
```suggestion
application pod for mutual TLS. You can find more info in the [Identity and certificate management section](#pki).
```
